firefox-install-demo.toml

```diff
@@ -33,9 +40,46 @@ select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
 type = "scalar"
 
 [metrics.install_success_rate.statistics]
-sum = {}
 mean = {}
 
+[metrics.install_volume_by_os_version]
+data_source = "firefox_installs"
+select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
+type = "scalar"
+
+[metrics.install_volume_by_os_version.statistics]
+sum = {}
+
+[metrics.install_volume_total]
+data_source = "firefox_installs_all_versions"
+select_expression = "IF(LOGICAL_OR(succeeded), 1, 0) * 100"
+type = "scalar"
+
+[metrics.install_volume_total.statistics]
+sum = {}
+
+[alerts]
+
+[alerts.install_success_rate]
+type = "threshold"
+metrics = [
+    "install_success_rate"
+]
+# is this per os version, or average over all, or....?
+min = [95]
+
+[alerts.install_volume_total]
+type = "avg_diff"
+metrics = [
+    "install_volume_total"
+]
+# is this a rolling window, eg: comparing monday -> sunday to monday -> tuesday
+# or comparing the entire previous week?
+window_size = 7
+# does this mean a drop of 20% will trigger an alert (eg: 100 -> 80), or a drop of 80% (eg: 100 -> 20)?
+max_relative_change = 0.8
+# percentile omitted because it doesn't apply here? Is that valid?
+
```

Comment:
```suggestion
```
Since totals aren't supported currently, this won't work. But to answer your questions:

> is this a rolling window, eg: comparing monday -> sunday to monday -> tuesday or comparing the entire previous week?

this compares the entire previous week, to the current week. For example, the average of the values from 2022-09-26 to 2022-10-02 get compared to the average of the values from 2022-10-03 to 2022-10-09

> does this mean a drop of 20% will trigger an alert (eg: 100 -> 80), or a drop of 80% (eg: 100 -> 20)?

in this case a drop of 80% would trigger the alert

> percentile omitted because it doesn't apply here? Is that valid?

correct